cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -121,7 +127,14 @@ func NewCMServer() *CMServer {
 // of volume.VolumeConfig which are then passed to the appropriate plugin. The ControllerManager binary is the only
 // part of the code which knows what plugins are supported and which CLI flags correspond to each plugin.
 type VolumeConfigFlags struct {
-	PersistentVolumeRecyclerTimeoutNFS int
+	// the pod to use as the default recycler.  default provided by the binary and the filepath flag can override.
+	PersistentVolumeRecyclerPodTemplate *api.Pod
```

Comment:
Each plugin needs the template to base its recycler from.  The plugin would override the fields it needs to.

I agree it's weird having both the filepath *and* the pod (from that file) in the same struct.

Thoughts on how that can be less weird?